Alignment/OfflineValidation/test/testingScripts/test_unitPixBary.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+function die { echo $1: status $2 ; exit $2; }
+
+runrange=376370-379254
+
+thistest="Alignment/PixelBarycentre single yaml configuration unitTest (test 1/3)"
+echo "TESTING $thistest..."
+pushd test_yaml/PixBary/single/testSinglePixBary/unitTest/$runrange
+./cmsRun validation_cfg.py config=validation.json || die "Failure running $thistest" $?
+popd
+
+thistest="Alignment/PixelBarycentre single yaml configuration mp3619 (test 2/3)"
+echo "TESTING $thistest..."
+pushd test_yaml/PixBary/single/testSinglePixBary/mp3619/$runrange
+./cmsRun validation_cfg.py config=validation.json || die "Failure running $thistest" $?
+popd
+
+thistest="Alignment/PixelBarycentre single json configuration unitTest (test 3/3)"
+echo "TESTING $thistest..."
+pushd test_json/PixBary/single/testSinglePixBary/unitTest/$runrange
```

Comment:
Here I am not sure if this test would bring anything new besides testing ability of All-in-one to create job directory based on the global config in json format (while yaml format is already tested above). But I don't have a strong opinion on this, perhaps the more the better.